pages/web-literacy/skillsdata.js

```diff
@@ -0,0 +1,131 @@
+var search = "Search";
```

Comment:
Are these the same as in `pages/web-literacy/activitydata.js`? That needs some DRYing out